### ISSUE TITLE
Cannot get list of discovered printers

### ISSUE THREAD

**Issue by melix99** [OpenPrinting/cpdb-backend-cups#25]
Hi! I was testing the new cpdb backend of the GTK printer dialog and I noticed a difference with the CUPS backend: the not-yet-setup printers do not show up in the printers list, while they do in the CUPS one. I tested this with the ippserver tool available in [ippsample](https://github.com/istopwg/ippsample).

Not sure if this is actually a cpdb/cups bug or if this should be handled in GTK code instead.

**Comment by TinyTrebuchet** [comment]
Are those printers listed when you use the `cpdb-text-frontend` utility (available with cpdb v2.0b4) ? If yes, then it's problem with the GTK code. If no, then it's a cpdb bug.

**Comment by melix99** [comment]
> Are those printers listed when you use the cpdb-text-frontend utility?

Nope. It shows up with `ippfind` though.

BTW I'm only testing with the `ippserver` utility, I don't have an actual ipp printer to test yet, but I don't think it matters much.

**Comment by TinyTrebuchet** [comment]
I am not able to reproduce this. Is `cups-browsed` running? What exact command are you using with the `ippserver` utility?

Also, can you manually start the CPDB CUPS backend and the cpdb-text-frontend utility with debugging enabled and post the output here?
```
CPDB_DEBUG_LEVEL=debug /usr/lib/print-backends/cups
```
```
CPDB_DEBUG_LEVEL=debug cpdb-text-frontend
```

**Comment by melix99** [comment]
Well, nevermind, it seems that the problem is that I was testing those things in a toolbox container and I guess something is wrong with that. I tested again in host and now it works correctly. I should have pointed that out first, sorry about that.

Anyway, when I start the cups backend manually (on toolbox), `cpdb-text-frontend` actually prints the printer. Also, I just noticed this error when I execute `cpdb-text-frontend` (without manually starting the backend):

```
[Error] [Frontend] Error getting CUPS printer list : GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable
```

No idea if it's actually a toolbox limitation or something else. Anyway I'll close this issue since it doesn't happen in host.
